image_width: 160
image_height: 120
camera_name: camera_link
camera_matrix:
  rows: 3
  cols: 3
  data: [95.34061672028389, 0.0, 80.5, 0.0, 95.34061672028389, 60.5, 0.0, 0.0, 1.0,]
distortion_model: plumb_bob
distortion_coefficients: 
  rows: 1
  cols: 5
  data: [0, 0, 0, 0, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [95.34061672028389, 0.0, 80.5, -6.673843170419873, 0.0, 95.34061672028389, 60.5, 0.0, 0.0, 0.0, 1.0, 0.0]
